packages/runtime/container-runtime/src/opLifecycle/opGroupingManager.ts

```diff
@@ -49,6 +49,37 @@ export class OpGroupingManager {
 		this.logger = createChildLogger({ logger, namespace: "OpGroupingManager" });
 	}
 
+	/**
+	 * Creates a new batch with a single message of type "groupedBatch" and empty contents.
+	 * @param referenceSequenceNumber - reference sequence number
+	 * @returns - IBatch with empty content
+	 */
+	public createEmptyGroupedBatch(
+		resubmittingBatchId: string,
+		referenceSequenceNumber: number,
+	): IBatch<[BatchMessage]> {
+		assert(
+			this.config.groupedBatchingEnabled,
+			"cannot create empty grouped batch when grouped batching is disabled",
+		);
+		const serializedContent = JSON.stringify({
```

Comment:
As discussed, let's put this assert back in here. It's better to fail fast here in this session than to commit to the op stream an unsupported op type.